spec/spec_helper.rb

```diff
@@ -45,7 +45,9 @@
   end
 
   config.before(:each) do |example|
-    DatabaseCleaner.strategy = :transaction
+    # use truncation to handle distinct read_slave db connection
+    # revert back to :transaction (faster) when the read slave requirement is removed
+    DatabaseCleaner.strategy = :truncation
```

Comment:
Can we do this with a tag on specific specs only? Tests are still running but I expect them to become at least twice as slow for the full suite (which is already taking a long time)?

Reply:
yeah for sure. 

Alternatively what do you think about using this gem instead https://github.com/kenn/slavery#transactional-fixtures

It seems to have better handling of connection switching, global kill switch (flipper?) and the ability to combine read slave / write master with nested blocks. It's a small dep that i've had a read through and the code seems ok.